### ISSUE TITLE
Cookies not sent

### ISSUE THREAD

**Issue by nmaier** [downthemall/downthemall#8]
We cannot fix it, only mozilla can, and they did, but only for Firefox 70+
https://bugzilla.mozilla.org/show_bug.cgi?id=1561654

So either wait for version 70 of Firefox, or get the Firefox Nighly

**Comment by nmaier** [comment]
Firefox 69 is released now, which supposedly fixes this.
I will keep this issue open a while longer so that users not on 69 yet can find this issue easier.

**Comment by blaueente** [comment]
I use Firefox 69 on Ubuntu. In my setup, cookies seem to work with "User defined settings"-> "Cookies from never visited sites", but are blocked with "Cookies from all third party sites" (sorry for imprecises translation, I have a german firefox, see below). Full Firefox restart required when settings are changed.

I can deal with malfunctioning websites for greater privacy, but my expectation would be that such a general privacy setting would at least not interfere with extensions, so I guess this is a Firefox bug? 

![grafik](https://user-images.githubusercontent.com/10434139/65550616-41f47c80-df20-11e9-9979-2091ec8296ae.png)

**Comment by nmaier** [comment]
Yep, I can reproduce this, and this is a firefox bug. The previous fix for the cookie behavior was only partial and breaks as soon as you configure the browser to block third party cookies.

I filed: https://bugzilla.mozilla.org/show_bug.cgi?id=1583715

**Comment by nmaier** [comment]
^ @blaueente
